python/runtime/common/py_NoiseModel.cpp

```diff
@@ -80,33 +93,51 @@ void bindNoiseModel(py::module &mod) {
              }
              return kraus_channel(kops);
            }),
-           "Create a KrausChannel given a list of KrausOperators.")
+           """
+           Create a KrausChannel given a list of KrausOperators.
```

Comment:
In these cases where the string really isn't all that long, sticking with the `"<string literal>"` syntax seems preferred to using raw strings.